domain/model/state/state.go

```diff
@@ -1680,3 +1680,9 @@ AND    ot.type = $dbPermission.object_type
 	}
 	return nil
 }
+
```

Comment:
Since we have `modelstate.go` this file should probably be renamed to `controllerstate.go`